docs/flags.md

```diff
@@ -0,0 +1,66 @@
+## Flags
+KICS scan flags are defined on `flags.json`, that can be found on `console/assets` directory.
+
+The structure of a flag on JSON definition should use the following structure:
+```json
+{
+    "flagName": {
+        "flagType": "<multiStr|str|bool|int>The type of the flag",
```

Comment:
What do you think about naming comma-separated string like this `[]string` instead of `multiStr`:

```suggestion
        "flagType": "<[]string|str|bool|int>The type of the flag",
```